src/utils/hints/oracles.js

```diff
@@ -0,0 +1,6 @@
+export const oraclesHints = {
+  oracleId: 'Unique Oracle identifier inherited from the account address of the creator.',
+  oracleRegistered: 'Date, time, and block height when the Oracle has been registered via OracleRegisterTx.',
```

Comment:
also here, "Estimated date, time and block..."

Reply:
You are right, fixed